spring-boot-project/spring-boot/src/test/java/org/springframework/boot/convert/StringToPeriodConverterTests.java

```diff
@@ -57,7 +57,6 @@ void convertWhenSimpleDaysShouldReturnPeriod(ConversionService conversionService
 
 	@ConversionServiceTest
 	void convertWhenSimpleWeeksShouldReturnPeriod(ConversionService conversionService) {
-		assertThat(convert(conversionService, "10w")).isEqualTo(Period.ofWeeks(10));
```

Comment:
I think this one should be kepts as it's testing lower-case `w` (the line below is testing upper-case `W`).

Reply:
@wilkinsona Thanks! I misread the code as a duplicate somehow 😓 

I reverted it.